app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,10 @@
+class FoodsController < ApplicationController
+  def index
+    @food = params[:q]
+    if @food.empty?
+      redirect_to root_path
+    else
+      @foods = FoodFacade.all_foods(@food)
+    end
+  end
```

Comment:
This is awesome. I really love seeing that you've accounted for this edgecase here.